### DISCUSSION TITLE
API for intergration

### DISCUSSION THREAD

**Discussion by Quorafind** [tadashi-aikawa/obsidian-another-quick-switcher#138] [Ideas]
Is it possible to exposed api for get files and suggester, I am willing to support same features in [Surfing](https://github.com/Quorafind/Obsidian-Surfing) but dont want to rebuild another quick switcher again.

**Comment by tadashi-aikawa** [comment]
@Quorafind 
Sorry for the late answer 🙇 

For now, I am not going to expose such an API.

Best regards.
